src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -2321,16 +2324,17 @@ test_bypass_spawning_via_mongocryptdBypassSpawn (void *unused)
 
    /* Create a MongoClient with encryption enabled */
    client_encrypted = test_framework_new_default_client ();
-   extra =
-      BCON_NEW ("mongocryptdBypassSpawn",
+   extra = BCON_NEW ("mongocryptdBypassSpawn",
```

Comment:
Missing ClangFormat changes.

Reply:
What do you mean?

Reply:
Lines immediately following line 2327 are not indented to match recent changes.

Reply:
Oh gross.

I have my VSCode set to hide whitespace in the working diff. When I selected the range and and staged it, it must have not staged the whitespace changes because they were hidden.